docs-src/go/how-to-backfill-a-schedule-in-go.md

```diff
@@ -4,10 +4,11 @@ title: How to backfill a Schedule in Go
 sidebar_label: Backfill Schedule
 description: To backfill a Schedule in Go, use `Backfill()` on `ScheduleHandle`.
 tags:
-- go sdk
-- code sample
-- schedule
-- backfill
+  - go sdk
+  - how-to-doc-type
+  - developer-guide-doc-type
+  - schedule
+  - backfill
```

Comment:
Where are these changes coming from?
Are these coming from running Assembly with `--samples` or are you manually editing this?

Reply:
That was me trying to fix a merge conflict. I can run `--samples` to make sure this flies

Reply:
Ran `--samples`. All good now.